apps/users/templates/advocate_europe_users/profile.html

```diff
@@ -2,129 +2,143 @@
 {% load i18n %}
 
 {% block title %}
-{% trans "Profile" %} {{ view.user.username }}
+    {% trans "Profile" %} {{ view.user.username }}
 {% endblock %}
 
 
 {% block content %}
-<div class="profile">
```

Comment:
I probably used the IDE to reformat the code. If it is a big problem, I can change it!